src/TextTruncate/TextTruncate.tsx

```diff
@@ -72,6 +72,13 @@ export default class TextTruncate extends React.PureComponent<Props> {
     } = this.props;
     const { truncated } = this.state;
 
+    // If text is null or undefined, return immediately
+    if (!text) {
+      return (
+        <></>
```

Comment:
I would actually return `null` here since fragments are typically used to wrap other elements.

Reply:
I should add that this is actually the recommended way as opposed to my personal preference 😄 
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering

Reply:
For sure! I didn't know null was an acceptable return here, since it's for rendering. Thanks! :)